### PR TITLE
Linear probing in Java

### LINKED ISSUE #5464: Linear Probing in Hashing
## 💥 Proposal

(A clear and concise description of what the proposal is.)
When collision happens in hashing we can use Linear probing to avoid the collision. I would like to code the Linear probing in Java

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

### PR DESCRIPTION
**Pull request by 1vivekanand**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Linear probing in hashing

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixed #5464

### CODE REVIEW

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
- Use proper code Format 
- Remove extra lines of code and spaces
- Add sample I/O at the end of the Code
- Add Time and Space Complexity at the end of the Code   
- Add new line at the EOF
- Use proper display message how you are reading and writing outputs
- Add input-output prompts to make the program more interactive.

File: Java/other/linearProbing.java [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+Output 1:
+48 63 98 -1 -1 -1 -1 15 
+
+Sample input 2:
+10 6
+45 80 90 60 74 10
+
+Output 2:
+80 90 60 10 74 45 -1 -1 -1 -1
```
Comment (ankitaggarwal23) [inline Java/other/linearProbing.java]: Use proper display message how you are reading and writing outputs

Add input-output prompts to make the program more interactive.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: Java/other/linearProbing.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+10
+Enter array size :
+6
+Enter elements of array :
+45 80 90 60 74 10
+
+Output 2:
+80 90 60 10 74 45 -1 -1 -1 -1 
+ */
+//-1 shows cell is still empty
```
Comment (ankitaggarwal23) [inline Java/other/linearProbing.java]: Remove these parts one is enough

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Use proper alignment of code and braces

**Comment by 1vivekanand** [comment]
@ankitaggarwal23 please add label for the level of issue

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Use proper alignment of code and braces

**Comment by ankitaggarwal23** [comment]
> Use proper alignment of code and braces

Read the instructions mentioned on Gssoc discord yesterday. Do the changes as per that.

**Comment by 1vivekanand** [comment]
@ankitaggarwal23 if still there is problem with braces then please point out the line

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**